components/stack/stack.mdx

```diff
@@ -0,0 +1,101 @@
+import { Subtitle, ArgsTable } from '@storybook/addon-docs/blocks';
+import DtStack from './stack';
+
+# Stack
+
+<Subtitle>
+  Stack is a layout component used to group elements together and apply a space between them.
+</Subtitle>
+
+## Base Style
+
+Stack base style starts without spacing between elements, on column direction and `div` as HTML element.
+
+## Slots & Props
+
+<ArgsTable of={DtStack} />
+
+## Usage
+
+### Available Gap
+
+```js
+[0, 100, 200, 300, 400, 500, 600]
+```
+
+### Available Directions
+
+```js
+['column', 'column-reverse', 'row', 'row-reverse']
+```
+
+### Available Breakpoints
+
+```js
+{ prefix: 'sm', mediaQuery: '(max-width: 480px)' },
+{ prefix: 'md', mediaQuery: '(max-width: 640px)' },
+{ prefix: 'lg', mediaQuery: '(max-width: 980px)' },
+{ prefix: 'xl', mediaQuery: '(max-width: 1264px)' },
+```
```

Comment:
This is a bit confusing since there's no javascript object like this that actually exists. Probably can just remove it from the code tags and use a markdown table. Something like this:

```
| Prefix | Max Width |
| -- | ------ |
| sm | 480px  |
| md | 640px  |
| lg | 980px  |
| xl | 1264px |
```
Which renders like this

| Prefix | Max Width |
| -- | ------ |
| sm | 480px  |
| md | 640px  |
| lg | 980px  |
| xl | 1264px |